cmd/provenanced/config/manager_test.go

```diff
@@ -153,3 +156,57 @@ func (s *ConfigManagerTestSuite) TestPackedConfigCosmosLoadGlobalLabels() {
 		s.Assert().Equal(appConfig.Telemetry.GlobalLabels, appConfig2.Telemetry.GlobalLabels)
 	})
 }
+
+func (s *ConfigManagerTestSuite) TestUnmanagedConfig() {
+	s.T().Run("unmanaged config is read with no other config files", func(t *testing.T) {
+		dCmd := s.makeDummyCmd()
+		configDir := GetFullPathToConfigDir(dCmd)
+		uFile := GetFullPathToUnmanagedConf(dCmd)
+		require.NoError(t, os.MkdirAll(configDir, 0o755), "making config dir")
+		require.NoError(t, os.WriteFile(uFile, []byte("banana = \"bananas\"\n"), 0o644), "writing unmanaged config")
+		require.NoError(t, LoadConfigFromFiles(dCmd))
+		ctx := client.GetClientContextFromCmd(dCmd)
+		vpr := ctx.Viper
+		actual := vpr.GetString("banana")
+		assert.Equal(t, "bananas", actual, "unmanaged field value")
+	})
+
+	s.T().Run("unmanaged config entry does not override other config", func(t *testing.T) {
+		dCmd := s.makeDummyCmd()
+		configDir := GetFullPathToConfigDir(dCmd)
+		uFile := GetFullPathToUnmanagedConf(dCmd)
+		require.NoError(t, os.MkdirAll(configDir, 0o755), "making config dir")
+		require.NoError(t, os.WriteFile(uFile, []byte("db_backend = \"still bananas\"\n"), 0o644), "writing unmanaged config")
+		require.NoError(t, LoadConfigFromFiles(dCmd))
+		ctx := client.GetClientContextFromCmd(dCmd)
+		vpr := ctx.Viper
+		actual := vpr.GetString("db_backend")
+		assert.NotEqual(t, "still bananas", actual, "unmanaged field value")
+		assert.Equal(t, tmconfig.DefaultConfig().DBBackend, actual, "unmanaged field default value")
+	})
+
+	s.T().Run("unmanaged config is read with unpacked files", func(t *testing.T) {
+		dCmd := s.makeDummyCmd()
+		uFile := GetFullPathToUnmanagedConf(dCmd)
+		SaveConfigs(dCmd, serverconfig.DefaultConfig(), tmconfig.DefaultConfig(), DefaultClientConfig(), false)
+		require.NoError(t, os.WriteFile(uFile, []byte("my-custom-entry = \"stuff\"\n"), 0o644), "writing unmanaged config")
+		require.NoError(t, LoadConfigFromFiles(dCmd))
+		ctx := client.GetClientContextFromCmd(dCmd)
+		vpr := ctx.Viper
+		actual := vpr.GetString("my-custom-entry")
+		assert.Equal(t, "stuff", actual, "unmanaged field value")
+	})
+
+	s.T().Run("unmanaged config is read with packed config", func(t *testing.T) {
+		dCmd := s.makeDummyCmd()
+		uFile := GetFullPathToUnmanagedConf(dCmd)
+		SaveConfigs(dCmd, serverconfig.DefaultConfig(), tmconfig.DefaultConfig(), DefaultClientConfig(), false)
+		require.NoError(t, PackConfig(dCmd), "packing config")
+		require.NoError(t, os.WriteFile(uFile, []byte("other-custom-entry = 8\n"), 0o644), "writing unmanaged config")
+		require.NoError(t, LoadConfigFromFiles(dCmd))
+		ctx := client.GetClientContextFromCmd(dCmd)
+		vpr := ctx.Viper
+		actual := vpr.GetInt("other-custom-entry")
+		assert.Equal(t, 8, actual, "unmanaged field value")
+	})
+}
```

Comment:
I noticed a lot of missing test coverage.  Is that alright?  It is mostly just for the errors.

Reply:
seems like those coverage flags are only triggered due to the error serialization fix (encode as error vs encode as value)